solarkraft/src/fetcher/callDecoder.ts

```diff
@@ -80,9 +81,16 @@ export async function extractContractCall(
     // https://developers.stellar.org/network/horizon/api-reference/resources/transactions/object
     const height = tx.ledger_attr
 
+    // Since Horizon does not return tx.result_meta_xdr anymore, make another query with getTransaction
+    const txDetails = await rpcServer.getTransaction(txHash)
+    if (txDetails.status == rpc.Api.GetTransactionStatus.NOT_FOUND) {
```

Comment:
Does this every happen? 
Feels like we should output a warning or something if we have to skip a tx that's expected to exist.

Reply:
This should not happen. However, a missing transaction does not have a field. Let's issue a warning